### DISCUSSION TITLE
Using a custom replication.policy.separator with DefaultReplicationPolicy

### DISCUSSION THREAD

**Discussion by lenin-joseph** [strimzi/strimzi-kafka-operator#10051] [Q&A]
Hi, 

In the case of bidirectional replication using mm2, when we tried using a custom replication.policy.separator( ex: "-") with DefaultReplicationPolicy , we see cyclic replication of topics. Could you confirm whether it's mandatory to use a CustomReplicationPolicy whenever we want to use a separator other than a "." ?

Thank you, 
Lenin

**Comment by scholzj** [comment]
That is probably a question on Kafka rather than on Strimzi.
